### PR TITLE
currect output tickets from server and * in list projects

### PR DESCRIPTION
**Pull request by artem-starovoitov**
# PR Details

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: client/lib/agile/commands/tickets.rb [hunk 1]
```diff
@@ -12,11 +12,11 @@ def create(ticket)
     end
 
     desc "list", "Tickets list"
+    # :reek:FeatureEnvy
```
Comment (mirterious) [inline client/lib/agile/commands/tickets.rb]: Пофиксить никак нельзя?

Comment (artem-starovoitov) [reply client/lib/agile/commands/tickets.rb]: если бы можно было, то я бы пофиксил.

Comment (mirterious) [reply client/lib/agile/commands/tickets.rb]: Попробуй вынести вывод имен в private метод, куда ты будешь передавать твоё info
```
def puts_tickets(info)
    info.each { |ticket| puts ticket["name"] if ... }
end
```

Comment (mirterious) [reply client/lib/agile/commands/tickets.rb]: Не сработало)()(

**Review by nieisnuje — APPROVED** [review]

**Review by mirterious — APPROVED** [review]

File: client/lib/agile/commands/tickets.rb (hunk 1 above)
Comment (artem-starovoitov) [reply client/lib/agile/commands/tickets.rb]: сработало.)

**Pull request merged**